module/VuFind/src/VuFind/Controller/CoverController.php

```diff
@@ -115,6 +125,24 @@ protected function getImageParams()
         ];
     }
 
+    /**
+     * Is the provided URL included on the configured allow list?
+     *
+     * @param string $url URL to check
+     *
+     * @return bool
+     */
+    protected function proxyAllowedForUrl(string $url): bool
+    {
+        $host = parse_url($url, PHP_URL_HOST);
+        foreach ((array)($this->config['coverproxyAllowedHosts'] ?? $config) as $regEx) {
```

Comment:
Should that be `?? []`

Reply:
Indeed it should. Not sure what I was thinking there -- but I was working in a hurry before starting scheduled server maintenance, so I'm not surprised that I did _something_ silly! ;-)